src/Robo/Plugin/Commands/DevelopmentModeCommands.php

```diff
@@ -435,10 +435,26 @@ public function frontendDevEnable($siteDir = 'default', array $opts = ['yes|y' =
         $this->io()->title('enabling front-end development mode.');
         $this->say("copying settings.local.php and development.services.yml into sites/$siteDir.");
 
-        $result = $this->taskFilesystemStack()
-            ->copy("$this->drupalRoot/sites/example.settings.local.php", $devSettingsPath, true)
-            ->copy("$this->drupalRoot/sites/development.services.yml", $this->devServicesPath, true)
-            ->run();
+        // Copy the example local settings file.
```

Comment:
This is the current state. Do we like how this is handled so far?

<img width="869" alt="command" src="https://user-images.githubusercontent.com/1349906/140552241-f484e11e-83c3-4b1a-a924-68fac3998d8e.png">

The next question is how we want to handle D7 in general, as this is really indicative of a larger problem as seen my the next error with the `development.services.yml` file and the other code that attempts to do more D8+ changes.

Reply:
We will create a new D7 specific development command for this use case.